src/css/flags/audioplayer.less

```diff
@@ -42,6 +42,11 @@
             display: none;
         }
     }
+    &.jw-flag-user-inactive {
```

Comment:
@madikarizma @jw-kaurand @hghazzi 

Shouldn't `:not(.jw-flag-audio-player)`be used in `jw-flag-user-inactive`?